Stormshield/stormshield_module/endpoint_actions.py

```diff
@@ -0,0 +1,15 @@
+from stormshield_module.base import StormshieldAction
+
+base_url = "/agents/{id}/tasks"
+
+
+class EndpointAgentIsolationAction(StormshieldAction):
+    verb = "post"
+    endpoint = base_url + "/network-isolation"
+    query_parameters: list[str] = ["forceServerIsolation", "comment"]
```

Comment:
These fields must be passed in the request body; not as query parameters
```suggestion
    query_parameters: list[str] = []
```